src/main/scala/fs2/Chunk.scala

```diff
@@ -29,6 +29,16 @@ trait Chunk[+A] { self =>
     iterator.map(f).copyToBuffer(buf)
     Chunk.indexedSeq(buf)
   }
+  /**
+   * `[[scanLeft]]` is a helper for process1.scan; it differs
+   * from standard collection behavior by not including
+   * the initial value in the output.
```

Comment:
It is weird to have a `scanLeft` function on any type that doesn't start with the seed value, so I'd say include it regardless. Relying on `drop` being fast for `Chunk` is totally fine.